vignettes/geotargets.Rmd

```diff
@@ -41,15 +41,21 @@ In this vignette we will demonstrate the use of the `terra` R package, and we wi
 -   `tar_terra_sprc()`
 -   `tar_terra_sds()`
 
+# How to run targets examples from vignettes
+
+The way targets typically works is you write a file named `_targets.R`, which describes the pipeline. See for example, the [`_targets.R` file in the demo-geotargets repo](https://github.com/njtierney/demo-geotargets/blob/main/_targets.R).
```

Comment:
Recommend using sentence breaks for easier diffs.  You can set this in RStudio options, for example.
```suggestion
The way targets typically works is you write a file named `_targets.R`, which describes the pipeline. 
See for example, the [`_targets.R` file in the demo-geotargets repo](https://github.com/njtierney/demo-geotargets/blob/main/_targets.R).
```

Reply:
I didn't know we could do this! I can't see how to change this option, I'll send you a message on slack.